src/components/general/ErrorMessage/styles.less

```diff
@@ -9,7 +9,7 @@
 
     .messageContainer {
         position: relative;
-        top: 0; left: 0; bottom: 0; right: 0;
+        top: 20%; left: 0; bottom: 0; right: 0;
```

Comment:
Does this have to be percentage?

Reply:
A but unsure how to solve this. Discussed with Lotte and concluded that the regular error message should always have a 120 px margin from the top. The small message should always be centered. 

Could this be defined in the component like i did in the most recent update? Or should this be left to the user of the component